hotline/pages/views.py

```diff
@@ -0,0 +1,66 @@
+from arcutils import will_be_deleted_with
+from django.shortcuts import render, get_object_or_404, redirect
+from django.contrib import messages
+from django.contrib.flatpages.models import FlatPage
+
+from hotline.pages.forms import FlatterPageForm
+
+from .perms import permissions
+
+
+def list_(request):
+    """
+
+    """
+    pages = FlatPage.objects.all()
+
+    return render(request, "pages/list.html", {
+        'pages': pages,
+    })
+
+
+@permissions.can_modify_page
+def create(request):
```

Comment:
The pattern you see in my code where I have these `create` and `edit` views, that defer to an `_edit` view is because different permission decorators are (usually) necessary on create and edit views.

In this case, because the same permission decorator applies to create and edit, you can just have a single view with `@permissions.can_modify_page`.